### PR TITLE
Enter answers timer

### PR DESCRIPTION
**Pull request by JamieZafar**

### CODE REVIEW

File: src/client/game/game-reducer.js [hunk 1]
```diff
@@ -1,15 +1,15 @@
-
 import * as actionTypes from './game-actions';
 
 const initialState = {
 	started: false,
-	leader: {
+    leader: {
         id: undefined,
```
Comment (kenneth-gray) [inline src/client/game/game-reducer.js]: If you want to keep these, maybe go for null instead unless you're checking explicitly for undefined?

File: src/client/game/game-reducer.js [hunk 2]
```diff
@@ -50,6 +51,36 @@ const game = (state = initialState, action) => {
             return Object.assign({}, state, {
                 disableVowel: action.payload
             });
+        case actionTypes.GET_CONSONANT: {
+            let letterList = Array.from(state.letterList);
+            letterList.push(action.payload);
+            return Object.assign({}, state, {
+                letterList: letterList
```
Comment (kenneth-gray) [inline src/client/game/game-reducer.js]: `[...state.letterList, action.payload]`

File: src/client/game/game-reducer.js [hunk 3]
```diff
@@ -50,6 +51,36 @@ const game = (state = initialState, action) => {
...
+            });
+        }
+        case actionTypes.GET_VOWEL: {
+            let letterList = Array.from(state.letterList);
+            letterList.push(action.payload);
+            return Object.assign({}, state, {
+                letterList: letterList
```
Comment (kenneth-gray) [inline src/client/game/game-reducer.js]: See above

File: src/client/game/game-reducer.js [hunk 4]
```diff
@@ -50,6 +51,36 @@ const game = (state = initialState, action) => {
...
+            });
+        }
+        case actionTypes.DISABLE_CONSONANT:
+            return Object.assign({}, state, {
```
Comment (kenneth-gray) [inline src/client/game/game-reducer.js]: Object.assign is used everywhere with that empty object - util time?

File: src/client/game/game.js [hunk 5]
```diff
@@ -34,18 +34,22 @@ export class Game extends Component {
                 <div>
                     {this.props.letterList}
                 </div>
-                <AnswerInput />
+                <div>
+                    {this.props.answering}
```
Comment (kenneth-gray) [inline src/client/game/game.js]: Do you want to output true or false here?

File: src/client/game/game-reducer.js (hunk 3 above)
Comment (JamieZafar) [inline src/client/game/game-reducer.js]: think we reverted these changes during the rebase we'll redo them now

File: src/client/services/cloak-service.js [hunk 6]
```diff
@@ -114,3 +122,35 @@ function messageGetConsonant() {
...
+
+function setDisableVowelDispatch(dispatch) {
+    disableVowelDispatch = dispatch;
+}
+
+function setStartAnsweringDispatch(dispatch) {
+    startAnsweringDispatch = dispatch;
+}
+
+function setStopAnsweringDispatch(dispatch) {
```
Comment (kenneth-gray) [inline src/client/services/cloak-service.js]: I thought these were going? Has this been rebased?

File: src/server/cloak-server.spec.js [hunk 7]
```diff
@@ -481,4 +481,4 @@ describe('cloak server', function() {
             expect(room.data.letterList.disableVowel).toEqual(true);
         });
     });
-});
+});
```
Comment (kenneth-gray) [inline src/server/cloak-server.spec.js]: Definitely wanted to see some tests - but maybe a little more than a newline? :stuck_out_tongue:

**Comment by JamieZafar** [comment]
:ship: dat :hankey: ?

File: src/server/cloak-server.spec.js (hunk 7 above)
Comment (JamieZafar) [inline src/server/cloak-server.spec.js]: checkListLength has just been moved - that is all (Kenny)

**Pull request merged**